build.sbt

```diff
@@ -165,3 +165,45 @@ lazy val nativeTest = Project(nativeTestId, file("nativeTest")).enablePlugins(Sc
     notPublish
   )
   .dependsOn(iterateeNative)
+
+lazy val site = Project(
+  id = "site",
+  base = file("site")
+).settings(
+  standardSettings,
+  name := "scalaz-site",
+  notPublish,
+  scalacOptions in (Compile, compile) -= "-Yno-adapted-args",
+  buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
+  buildInfoPackage := "scalaz",
+  buildInfoObject := "BuildInfo"
```

Comment:
https://github.com/scalaz/scalaz/blob/f6dd5e43dd695821a579367a62fe1cb0ecbd13dc/project/build.scala#L350-L352

We can use `ScalazBuildInfo` in scalaz-core.